uk_bin_collection/uk_bin_collection/councils/RoyalBoroughofGreenwich.py

```
@@ -65,6 +66,29 @@ def parse_data(self, page: str, **kwargs) -> dict:
+        holiday_response.raise_for_status()
+        soup = BeautifulSoup(holiday_response.text, features="html.parser")
+        soup.prettify()
+
+        holiday_dict = {}
+
+        table = soup.find("table", {"class": "tablesaw tablesaw-stack"})
+        if table:
+            rows = table.find("tbody").find_all("tr")
+            for row in rows:
+                original_collection_date = row.find_all("td")[0].get_text(strip=True)
+                original_collection_date = re.findall("\w+ \d+ \w+", original_collection_date)[0]
```

Ruff (0.15.12)

[warning] Invalid escape sequence: `\w`

Use a raw string literal

(W605)